main.tf

```diff
@@ -95,7 +95,7 @@ resource "aws_instance" "default" {
 resource "aws_eip" "default" {
   count    = local.eip_enabled ? 1 : 0
   instance = join("", aws_instance.default.*.id)
-  vpc      = true
+  domain   = "vpc"
```

Comment:
What version of the aws provider deprecated this?

Whatever the version, please update the versions.tf file to use a >= for that version and please copy that update to  examples/complete/versions.tf

Reply:
Hi @nitrocode , thanks for your quick response. I've realized that we don't need to explicitly set either `vpc` or `domain` since we're using the default values. Simply omitting them will ensure maximum compatibility, so no updates to dependencies are necessary.